packages/eslint/src/migrations/update-17-1-0/update-typescript-eslint.spec.ts

```diff
@@ -84,7 +84,6 @@ export const rule = ESLintUtils.RuleCreator(() => __filename)({
     type: 'problem',
     docs: {
       description: \`\`,
-      recommended: 'error',
```

Comment:
This was an old migration, I don't think it technically matters if it's updated or not?

Reply:
It didn't ever serve any purpose so it's just setting people up for an issue doing the road if they run the migration right now